documentation/src/docs/pravega-write-read-methods.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Methods for Writing and Reading Pravega Streams
+---
+
+import { IfHaveFeature, IfMissingFeature, SDPVersion } from 'nautilus-docs';
+
+## Overview
+
+This table describes the methods that can be used to write and read data in Pravega streams. It is intended to be a comprehensive list which includes open-source, experimental, and commercial products that integrate with Pravega.
+
+Method                                                                                | Write | Transactional Write | Read | Special Features | Notes
```

Comment:
We could add:

**Debezium**
https://debezium.io/documentation/reference/operations/debezium-server.html#_pravega
Write: Yes
Transactional Write: Yes
Read: No

Reply:
*ZIO*
https://github.com/cheleb/zio-pravega
Write: Yes
Transactional Write: No
Read: Yes